marklogic-data-hub-central/ui/src/pages/Run.tsx

```diff
@@ -92,13 +93,15 @@ const Run = (props) => {
   const [runStarted, setRunStarted] = useState<any>({});
   const [runEnded, setRunEnded] = useState<any>({});
   const [running, setRunning] = useState<any[]>([]);
+  const [isStepRunning, setIsStepRunning] = useState(false);
   const [uploadError, setUploadError] = useState("");
   const [openJobResponse, setOpenJobResponse] = useState<boolean>(false);
   const [jobId, setJobId] = useState<string>("");
 
   const [successModal, setSuccessModal] = useState<successModalType>({...defaultSuccessModal});
   const [errorModal, setErrorModal] = useState<errorModalType>({...defaultErrorModal});
   const [failedModal, setFailedModal] = useState<failedModalType>({...defaultFailedModal});
+  //const [successModalMultipleSteps, setSuccessModalMultipleSteps] = useState(false);
```

Comment:
Clean unused code

Reply:
Just in case will remove this in the next PR related with this, perhaps we will need this.